ledger-tool/src/bigtable.rs

```diff
@@ -462,7 +471,7 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .value_name("LIMIT")
                                 .validator(is_slot)
                                 .index(2)
-                                .default_value("1000")
+                                .default_value("18446744073709551615")
```

Comment:
Just noticed: the `about` for this subcommand says: "Show historical transactions affecting the given address, ordered based on the slot in which they were confirmed in from lowest to highest slot"

Looks like it's copied from cli/cluster_query.rs, but needs updating there since #11259